code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -322,6 +367,17 @@
 	moduleselect_icon = "medical"
 	can_be_pushed = FALSE
 	hat_offset = 3
+	borg_skins = list(
+		"Default" = list(SKIN_ICON_STATE = "medical", SKIN_TRANSFORM = TRUE),
+		"Basic" = list(SKIN_ICON = 'icons/mob/robots_extra.dmi', SKIN_ICON_STATE = "robot_medical", SKIN_LIGHT_KEY = "robot"),
```

Comment:
just make SKIN_ICON file static, I don't think you used any different files

Reply:
In the code, icon is already set to use robots.dmi, SKIN_ICON_STATE overrides it to allow loading of icons from different files, which it does here to keep it easier to maintain in an orderly manner, plus future support for loading separate icon files.